modules/service/src/main/scala/lucuma/odb/graphql/topic/OdbTopic.scala

```diff
@@ -49,11 +49,11 @@ object OdbTopic:
   )(update: PartialFunction[Array[String], Option[U]]): OdbTopic[E] =
     new OdbTopic[E]:
 
-      def updates[F[_]: Logger](
+      def updates[F[_]: Concurrent: Logger](
         s:         Session[F],
         maxQueued: Int
       ): Stream[F, U] =
-        s.channel(channel).listen(maxQueued).flatMap: n =>
+        s.channel(channel).listen(maxQueued).prefetchN[F](1024).flatMap: n =>
```

Comment:
I have an issue here.  When the world is updated and the calculator has to run through all the observations there will be 1000s of events.  When the number of events surpasses `maxQueued` then the channel seems to hang and I can't shut the service down cleanly.  I added `prefetch` here and use `concurrently` instead of `merge` to combine the streams, but these only mitigate the problem.  For now I've just increased the `maxQueued` value to be greater than the number of observations we have.